packages/css-data/bin/mdn-data.ts

```diff
@@ -2,9 +2,9 @@ import { definitionSyntax, type DSNode } from "css-tree";
 import properties from "mdn-data/css/properties.json";
 import units from "mdn-data/css/units.json";
 import syntaxes from "mdn-data/css/syntaxes.json";
-import type { StyleValue } from "../src/css";
-import { popularityIndex } from "./popularity-index";
-
+//import type { StyleValue } from "../src/css";
+import { popularityIndex } from "../src/popularity-index";
+type StyleValue = any;
```

Comment:
will fix in the next PR, was moving types and discovered a rabbit whole of code duplication we got